### DISCUSSION TITLE
Understanding the detail in Flowviz throughput

### DISCUSSION THREAD

**Discussion by davejcory** [nbrown02/FlowViz#97] [General]
@nbrown02 I have just installed the Flowviz plugin and and reviewing the metrics produced along with your wiki.
On the Throughput metric you mention that cycle time is the time between the start line and the completed line.

Can you explain what constitutes the Start Line and the Completed Line please. Is it based on the ADO uber states of Proposed, In Progress and Completed or some other setting where we need to decide what our start and end points are?

Thanks in advance

**Comment by nbrown02** [comment]
Hi there,Yes that’s correct. If you check this link https://github.com/nbrown02/FlowViz/wiki/7.-FAQs#how-is-lead-timecycle-time-calculated in the FAQs that shows the state/state category mapping 🙂

**Reply by davejcory** [reply]
Ignore last comment. Just read the microsoft documentation where it confirmed cycle time is based on items being in the In Progress state.

**Comment by davejcory** [comment]
Ignore the last commment. I have just realised it is

**Comment by davejcory** [comment]
@nbrown02 Last one I hope and apologies if I have missed something. So a number of the projects have multiple teams associated with the single project. Is there a way to filter the metrics to report against a single team within the project rather than against all teams.  I'm guessing I could amend the base analytics view and change it to only select a specific team but then if I wanted to report against a second team separately then it would not work I'm guessing 

Have I missed something or is there a way to modify the configuration to either select a specific team within the PowerBI report, have I missed something during the initial configuration of Flowviz or is it just controlled at the analytics view of ADO.

Regards

**Comment by nbrown02** [comment]
Nope this is a very good question. I have created some custom versions of the template that filter by team or area path, links provided in the replies to this issue https://github.com/nbrown02/FlowViz/issues/93I would note that this isn’t a version I “support” if you find issues…just gives you hopefully the answer you’re looking for 🙂

**Reply by nbrown02** [reply]
@davejcory It is also possible to amend the queries to do multiple teams, you just need to copy/paste what I’ve done for a single team and add the additional team name(s) in. I don’t have an example to hand as away from my machine but it would be something like:

and (Teams/any(x:x/TeamName eq '{teamname}) or Teams/any(x:x/TeamName eq '{teamname}) or Teams/any(x:x/TeamName eq '{teamname})

**Reply by davejcory** [reply]
@nbrown02 thanks. When you said "done for a single team" is it not done for a single project rather than team? Either way, let me investigate and see what I can see. Thanks

**Reply by nbrown02** [reply]
> @nbrown02 thanks. When you said "done for a single team" is it not done for a single project rather than team? Either way, let me investigate and see what I can see. Thanks
> 
> 

The links I shared in the reply are for a single team within a project 🙂
